### ISSUE TITLE
Hyperlinks only work when window is scrolled to the bottom

### ISSUE THREAD

**Issue by ssbssa** [mintty/mintty#1021]
I use mintty 3.2.0 from MSYS2.

The [clickable hyperlinks](https://github.com/mintty/mintty/wiki/CtrlSeqs#hyperlinks) only work when the mintty window is scrolled to the absolute bottom.

E.g. with this:
`printf '\e]8;;http://example.com\e\\This is a link\e]8;;\e\\\n'`
I get the example.com link, and when hovering with CTRL pressed, it shows the URL in the title bar, and when clicked it opens it in my browser.

But if I scroll up (no matter how much), the link is still shown in the title bar, but when clicked I get an error dialog with this message:
```
---------------------------
Error
---------------------------
The system cannot find the file specified. 
C:\src\tests\This
---------------------------
OK   
---------------------------

```

In my tests just now I also noticed that if I scroll up 1 line, then when I click on the line above it opens the example.com link in the browser. And when I scroll up 2 lines, then the clickable link is 2 lines above the correct line, etc.

So when clicking, it actually always assumes that the window is scrolled to the bottom.

**Comment by mintty** [comment]
Uploaded a fix.

**Comment by ssbssa** [comment]
I just tested it, works now, thanks for the fast fix.

**Comment by mintty** [comment]
Released 3.3.0.
